src/index.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] Please don't disable eslint rules. This is one of the basic requirements.

Reply:
you don't need to disable this, as you've used the variable so ESLint won't give you an error